data/json/bionics.json

```diff
@@ -536,8 +536,11 @@
         "occupied_bodyparts": [
             [ "TORSO", 10 ]
         ],
-        "act_cost": 50,
-        "description": "You are equipped with bionic stimulators which augment your haematopoiesis system, allowing you to accelerate white blood cell production using bionic power."
+        "toggled": true,
+        "act_cost": 10,
+        "react_cost": 1,
+        "time": 600,
```

Comment:
This explanation makes sense, so only issue preventing this PR from merge is a typo mentioned above.